integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/jdbc/TestMultipleJdbcLoad.java

```diff
@@ -122,7 +122,7 @@ public void testCreateMultipleJdbc() throws URISyntaxException {
         postgreSqlCatalog
             .asSchemas()
             .listSchemas(Namespace.of(metalakeName, postgreSqlCatalogName));
-    Assertions.assertEquals(0, nameIdentifiers.length);
+    Assertions.assertNotEquals(0, nameIdentifiers.length);
```

Comment:
we should check the schemas only include "public" schema if nameIdentifiers.length > 0